product_docs/docs/migration_toolkit/55/installing/index.mdx

```diff
@@ -81,9 +81,9 @@ Select a link to access the applicable installation instructions:
 
 - [Debian 12](linux_arm64/mtk_debian_12)
 
-## Macintosh
+## Mac
 
-- [Mac OS X](macos)
+- [macOS](macos)
```

Comment:
We are no longer building macOS installers. References to macOS should be removed.